@ndlib/gatsby-theme-marble/src/utils/appUtils.js

```diff
@@ -165,15 +165,19 @@ const compilations = [
 
 // end mock data
 export const getUserCompilations = (userName) => {
-  return compilations.filter(compilation => {
-    return compilation.user.userName === userName
-  })
+  console.log(userName)
+  return []
+  // return compilations.filter(compilation => {
+  //   return compilation.user.userName === userName
+  // })
 }
 
 export const getCompilation = (compilationId) => {
-  return compilations.find(compilation => {
-    return compilation.id === compilationId
-  })
+  console.log(compilationId)
+  return null
+  // return compilations.find(compilation => {
+  //   return compilation.id === compilationId
+  // })
```

Comment:
`getUserCompilations` and `getCompilation` will likely get removed in the next PR related to User Portfolio API implementation.